packages/terra-icon/CHANGELOG.md

```diff
@@ -5,10 +5,8 @@
 ## 3.42.0 - (April 6, 2021)
 
 * Changed
-  * Removed unnecessary `wdio.conf.js` file
-  * Upgrade tests to terra-functional-testing and WebDriverIO v6
-  * Updated Jest command
-  * Updated screenshots with elements that are out of bound
+  * Bumped `one-cerner-style-icons` to v1.38.0
+  * Minor version bumps to dependencies
```

Comment:
I think there are a few conflicts here. Some items were removed and the new changes are in a previously release section

Reply:
```suggestion
  * Removed unnecessary `wdio.conf.js` file
  * Upgrade tests to terra-functional-testing and WebDriverIO v6
  * Updated Jest command
  * Updated screenshots with elements that are out of bound
```